### ISSUE TITLE
Uninitialized constant error in templates after upgrade

### ISSUE THREAD

**Issue by toncid** [ai/autoprefixer-rails#141]
Upgrading Autoprefixer from 8.6.5 to 9.1.3 yields a template error:

```
ActionView::Template::Error (uninitialized constant ExecJS::Runtimes::RubyRacerRuntime):
    11: 
    12:                 <%= content_tag :div, class: :'navbar-brand' do %>
    13:                     <%= link_to home_path do %>
    14:                         <%= image_tag 'logo.svg', height: 27 %>
    15:                     <% end %>
    16:                 <% end %>
    17: 
  app/assets/javascripts/admin/app-run.js.coffee.erb:340:in `_evaluate_template'
```

There are no upgrade instructions. Any advice?

**Comment by ai** [comment]
Try to add latest `execjs`

**Comment by RISCfuture** [comment]
Didn't work for me

**Comment by RISCfuture** [comment]
It comes from https://github.com/ai/autoprefixer-rails/blob/master/lib/autoprefixer-rails/processor.rb#L149

Seems like if the RubyRacer runtime is not installed, it makes sense that that constant would be undefined. Compare it using `to_s`?

**Comment by RISCfuture** [comment]
Ah no, I see the problem. `ExecJS.runtime` is an `ExecJS::RubyRacerRuntime`, not `ExecJS::Runtimes::RubyRacerRuntime`.

**Comment by toncid** [comment]
@ai Already at the latest `execjs` version: 2.7.0

**Comment by toncid** [comment]
It might be helpful to know that I am using a Rails v4.2 app, which is still supported.

**Comment by evgen3d** [comment]
Same problem for 9.1.3.
Latest release doesn't contains this fix yet? 

Thanks

**Comment by ai** [comment]
Sorry, it doesn't contain. I will try to release it on next week, but maybe it will be delayed to week after

**Comment by evgen3d** [comment]
Thank you very much

**Comment by ai** [comment]
Released in 9.1.4
